book/src/concepts/world.md

```diff
@@ -115,6 +115,8 @@ All:
     world.delete_all().expect("Failed to delete all entities.");
 ```
 
+__Note: Entities are lazily deleted, which means that deletion only happens at the end of the frame and not immediately when calling the `delete` method.__
```

Comment:
Just curious if the engine calls maintain here or if we want to kind of link this back to specs maintain? Just to complete the image of how specs integrates etc.

Reply:
yes, but I just removed it per request from a previous review

Reply:
I still see it in the latest commit?

Reply:
I'm sure I pushed it. https://github.com/amethyst/amethyst/pull/881/commits/984b3dbdce56d860d115cd1437b7be3c660ad5c1